specs/1118-ORDS-submit_order.md

```diff
@@ -12,3 +12,14 @@
 
 - I can see the raw offset price of the order (<a name="1118-ORDS-008" href="#1118-ORDS-008">1118-ORDS-008</a>)
 - I can see the reference price of the order (<a name="1118-ORDS-009" href="#1118-ORDS-009">1118-ORDS-009</a>)
+
+### Market Orders
+
+- I can see 'Market price' (<a name="1118-ORDS-013" href="#1118-ORDS-013">1118-ORDS-013</a>)
+
+### Enriched Orders
+
+- I see enriched data for the market (code) when data has been loaded appropriately (<a name="1118-ORDS-011" href="#1118-ORDS-011">)
+- I see enriched data for the order size when data has been loaded appropriately (<a name="1118-ORDS-012" href="#1118-ORDS-012">1118-ORDS-012</a>)
+- I see enriched data for the order price when data has been loaded appropriately (<a name="1118-ORDS-013" href="#1118-ORDS-013">1118-ORDS-013</a>)
```

Comment:
duplicate ac code, we already have 013. Not sure how you want to organise but yeah you have 013 above

Reply:
looks like my find and replace didn't catch things properly! good spot, thank you